### PR TITLE
Polishing app

### PR DESCRIPTION
**Pull request by testinnplayin**
Hi Jack,
Is this alright?

### CODE REVIEW

File: ideas/dropdown.js [hunk 1]
```diff
@@ -0,0 +1,38 @@
...
+// example for one button
+
+function handleNavClick() {
+	$('.js-dropdown-btn-classes').click(function(e) {
+		e.preventDefault();
+
+		$('.js-actions-dropdown-classes').toggleClass('show');
+	});
+
+	window.onclick = function(e) {
```
Comment (mlnck) [inline ideas/dropdown.js:12]: Should not need if we bind the above with "on":
$('.js-dropdown-btn-classes').on('click',function(e)....)

File: ideas/dropdown.js [hunk 2]
```diff
@@ -0,0 +1,38 @@
...
+}
+
+function handleMoreClick() {
+	$('.js-dropdown-btn-1').click(function(e) {
+		e.preventDefault();
+
+		$('.js-actions-dropdown-1').toggleClass('show');
+	});
+
+	window.onclick = function(e) {
```
Comment (mlnck) [inline ideas/dropdown.js:26]: Same as above

File: public/index.html [hunk 3]
```diff
@@ -5,52 +5,102 @@
...
+					</ul>
+					<form id="search-form">
+						<label for="search" aria-label="search" aria-hidden="true" class="screen-reader">Search:</label>
+						<input type="text" name="search" id="search" placeholder="Search" />
+						<button type="submit"><i class="fa fa-search" aria-hidden="true"></i></button>
+					</form>
 				</div>
-			</nav>	
+			</div>
 		</header>
```
Comment (mlnck) [inline public/index.html:70]: Is the nav element closed out?
(Opened on line 15)

File: public/scripts/app.js [hunk 4]
```diff
@@ -36,61 +45,145 @@ function setCurrentSpan(ele, view) {
...
+}
+
+function handleDropDownClick(index) {
+	$('.js-dropdown-btn-' + index).click(function(e) {
+		e.preventDefault();
+
+		$('.js-actions-dropdown-' + index).toggleClass('show');
+	});
+
+	$(window).click(function(e) {
```
Comment (mlnck) [inline public/scripts/app.js:129]: Same comment as above with using ".on"

File: public/scripts/app.js [hunk 5]
```diff
@@ -163,19 +256,21 @@ function drawBodyRows(klasses) {
...
 		$('.js-send-to-del-trigger').val(value);
 	}
 }
 
 function drawHeadRow(klasses) {
-	let arr = ['Class Name', 'Subject', 'Grade Level', 'Term'],
+	let arr = ['Class', 'Subject', 'Grade', 'Term'],
 		lng = arr.length;
 
 	for (let i = 0; i < lng; i++) {
```
Comment (mlnck) [inline public/scripts/app.js:272]: Not a huge difference...but consider map
`arr.map(function(itm){ /*lines 273,274*/ })`

File: public/scripts/create-class.js [hunk 6]
```diff
@@ -1,5 +1,19 @@
 'use strict';
 
+function handleDropDownClickClassCreate(index) {
+	$('.js-dropdown-btn-' + index).click(function(e) {
+		e.preventDefault();
+
+		$('.js-actions-dropdown-' + index).toggleClass('show');
+	});
+
+	$(window).click(function(e) {
```
Comment (mlnck) [inline public/scripts/create-class.js:10]: Same comment as above with ".on"

File: public/scripts/dropdown.js [hunk 7]
```diff
@@ -0,0 +1,23 @@
...
+	let arr = ['classes', 'students'];
+
+	arr.forEach(function(item) {
+		$('.js-dropdown-btn-' + item).click(function(e) {
+			e.preventDefault();
+
+			$('.js-actions-dropdown-' + item).toggleClass('show');
+		});
+
+		window.onclick = function(e) {
```
Comment (mlnck) [inline public/scripts/dropdown.js:15]: Same as above with ".on"

**Review by mlnck — APPROVED** [review]
We can walk through these comments today and then merge, push to Heroku, and submit 👍

File: ideas/dropdown.js (hunk 1 above)
Comment (testinnplayin) [reply ideas/dropdown.js:12]: Desired functionality: user clicking anywhere in window other than on dropdown menu should be able to close that menu

File: ideas/dropdown.js (hunk 2 above)
Comment (testinnplayin) [reply ideas/dropdown.js:26]: Desired functionality: user clicking anywhere in window other than on dropdown menu should be able to close that menu

File: public/index.html (hunk 3 above)
Comment (testinnplayin) [reply public/index.html:70]: Closed on line 45

File: public/scripts/app.js (hunk 4 above)
Comment (testinnplayin) [reply public/scripts/app.js:129]: Desired functionality: user clicking anywhere in window other than on dropdown menu should be able to close that menu

File: public/scripts/app.js (hunk 5 above)
Comment (testinnplayin) [reply public/scripts/app.js:272]: Ok will keep it in mind! Thanks

File: public/scripts/create-class.js (hunk 6 above)
Comment (testinnplayin) [reply public/scripts/create-class.js:10]: Desired functionality: user clicking anywhere in window other than on dropdown menu should be able to close that menu

File: public/scripts/dropdown.js (hunk 7 above)
Comment (testinnplayin) [reply public/scripts/dropdown.js:15]: Desired functionality: user clicking anywhere in window other than on dropdown menu should be able to close that menu

**Review by testinnplayin — COMMENTED** [review]
Applied comments

**Pull request merged**